include/ginkgo/core/base/lin_op.hpp

```diff
@@ -447,6 +455,9 @@ class WritableToMatrixData {
 
 /**
  * A LinOp implementing this interface can be preconditioned.
+ *
+ * @ingroup precond
```

Comment:
I might have missed something, but why is this in group `precond`?

Reply:
The interface is called `preconditionable`, so I think it should be linked to the preconditioners.

Reply:
Oh, my bad, I actually did not look at the name of the class itself...